### PR TITLE
add test for loading non-module, fixes #109

### LINKED ISSUE #109: Add console.log test case for when not loaded as a module
Current test case only validates the case when added as a module.

### PR DESCRIPTION
**Pull request by fippo**
@KaptenJansson -- I wonder if we should also test the intentionally globalized variables so we can modularize things without breaking stuff.

### CODE REVIEW

**Comment by fippo** [comment]
so presumably the missing return in the iceTransportPolicy test went unnoticed because it was the last test... very odd. And very hard to spot.

File: test/test.js [hunk 1]
```diff
@@ -766,3 +767,27 @@ test('iceTransportPolicy is translated to iceTransports', function(t) {
...
 });
+
+// This MUST to be the last test since it loads adapter
+// again which may result in unintended behaviour.
+test('Non-module logging to console still works', function(t) {
+  var logCount = 0;
+  var saveConsole = console.log.bind(console);
+  console.log = function() {
+    logCount++;
+    //saveConsole.apply(saveConsole, arguments);
```
Comment (KaptenJansson) [inline test/test.js]: Remove the commented code.

**Comment by KaptenJansson** [comment]
Nice test and catch of the missing return (I wish github web ui could display the diff in the whole file so one can see the stuff around + the new changes).

I think it makes sense to test everything that we expose globally, filed #47 a while back that touches on this.

**Comment by fippo** [comment]
checking for "public api" as well. Note that I did not consider webrtcUtils part of that.

**Comment by fippo** [comment]
not sure why FF stable didn't return @ https://travis-ci.org/webrtc/adapter/jobs/75443265 -- all tests pass?

**Comment by KaptenJansson** [comment]
That looks really odd, does not repro locally off course? ;)

**Comment by fippo** [comment]
let's try again and see if it persists...

**Comment by KaptenJansson** [comment]
Thanks for this! LGTM.

**Pull request merged**